gulpfile.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+
+// This file is only for generating the docs
+// No need to use any of this if working on the main website
+
+const { src, dest, series } = require('gulp');
+const pump = require('pump');
+const through2 = require('through2');
+const frontMatter = require('gray-matter');
+const download = require('github-download-directory');
+
+// Exports for task registration
+exports.default = series(fetchDocs, removeComments);
+
+const owner = 'gulpjs';
+const repo = 'gulp';
+const directory = 'docs';
+
+const fmOptions = {
+  delimiters: ['<!-- front-matter', '-->']
+};
+
+function fetchDocs() {
+  return download(owner, repo, directory, { sha: "wip-docs" });
+}
+
+function removeComments(cb) {
```

Comment:
Question Part 2: But here you are removing the comments? Docusaurus needs that data at the top of the markdown file to populate the sidebar. Am I misunderstanding this?

Reply:
@somethingnew71  it is actually replacing `<!-- front-matter` with `--` because I don't want the table showing up in the rendered markdown.